src/static/riot/analytics/analytics.tag

```diff
@@ -0,0 +1,540 @@
+
+
+    <div class="ui top attached tabular menu">
+        <a class="active item" data-tab="competitions">Competitions</a>
+        <a class="item" data-tab="submissions">Submissions</a>
+        <a class="item" data-tab="users">Users</a>
+    </div>
+
+        <div class="ui bottom attached active tab segment" data-tab="competitions">
+            <div class="ui statistics">
+                <div class="statistic">
+                    <div class="value">
+                        {competitions}
+                    </div>
+                    <div class="label">
+                        Competitions Created
+                    </div>
+                </div>
+                <div class="statistic">
+                    <div class="value">
+                        {competitions_published}
+                    </div>
+                    <div class="label">
+                        Competitions Published
+                    </div>
+                </div>
+            </div>
+            <div class='chart-container'>
+                <canvas ref="competition_chart"></canvas>
+            </div>
+        </div>
+
+        <div class="ui bottom attached tab segment" data-tab="submissions">
+            <div class="ui statistic">
+                <div class="value">
+                    {submissions_made}
+                </div>
+                <div class="label">
+                    Submissions Made
+                </div>
+            </div>
+            <div class='chart-container'>
+                <canvas ref="submission_chart"></canvas>
+            </div>
+        </div>
+
+        <div class="ui bottom attached tab segment" data-tab="users">
+            <div class="ui statistic">
+                <div class="value">
+                    {users_total}
+                </div>
+                <div class="label">
+                    Users Joined
+                </div>
+            </div>
+            <div class='chart-container'>
+                <canvas ref="user_chart"></canvas>
+            </div>
+        </div>
+
+        <button class="ui big blue button" onclick="{ download_analytics_data }">Download</button>
+    <script>
+        var self = this
+
+        /*---------------------------------------------------------------------
+         Init
+        ---------------------------------------------------------------------*/
+        self.errors = []
+
+        self.competitionsChart;
+        self.submissionsChart;
+        self.usersChart;
+        self.time_unit = 'month';
+
+        self.competitions_data;
+        self.submissions_data;
+        self.users_data;
+
+        self.start_date = {
+            year: 2018,
+            month:8,
+            day: 1,
+        }
+
+        var today = new Date()
+        self.end_date = {
+            year: today.getFullYear(),
+            month: today.getMonth() + 1,
+            day: today.getDate(),
+        }
+
+        self.convert_datestring_to_object = function (s, back_adjust) {
```

Comment:
may want to bring in moment.js to do this.

Reply:
Replaced date logic with Luxon where possible.